apps/server/test/lexical/server/project/node_test.exs

```diff
@@ -12,6 +12,8 @@ defmodule Lexical.Server.Project.NodeTest do
   setup do
     project = project()
 
+    {:ok, _} = start_supervised(Lexical.RemoteControl.ProjectNodeSupervisor)
```

Comment:
this would not be necessary if the project's supervisor had the project node supervisor in it.